### PR TITLE
[FEATURE] Add FLOAT cObject for JSON rendering

### PR DESCRIPTION
**Pull request by lochmueller**
Resolve https://github.com/TYPO3-Initiatives/headless/issues/381 and add a new FLOAT cObject to support float values in JSON.

### CODE REVIEW

**Comment by tmotyl** [comment]
@lochmueller Are you able to add unit tests for it?

File: Classes/ContentObject/FloatContentObject.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+use TYPO3\CMS\Frontend\ContentObject\AbstractContentObject;
+
+/**
+ * FLOAT Content Object only for use in JSON content object
+ *
+ * ** not working ** outside of JSON content object
+ */
+class FloatContentObject extends AbstractContentObject
+{
+    use LoggerAwareTrait;
```
Comment (tmotyl) [inline Classes/ContentObject/FloatContentObject.php:24]: what do we need this trait for?

Comment (lochmueller) [reply Classes/ContentObject/FloatContentObject.php:24]: Not needed, but it is just a copy of the INT cObject. Perhaps we check this in all classes after?!

**Comment by lochmueller** [comment]
@tmotyl dropped the useless traits in all three cObjects. There is no test currently (because INT and BOOL has also no direct Unit test ;) )

**Comment by twoldanski** [comment]
@lochmueller @tmotyl I think we can merge it, I have tests for it (I closed my PR for same feature)

**Review by twoldanski — DISMISSED** [review]

**Comment by lukaszuznanski** [comment]
@lochmueller please fix codestyle and we can merge it

**Comment by lochmueller** [comment]
@lukaszuznanski done.

**Review by lukaszuznanski — APPROVED** [review]

**Pull request merged**